lib/checks/report/report.rb

```diff
@@ -0,0 +1,28 @@
+module Checks
+  module Report
+    class Report
```

Comment:
That would be OK. I think we should address it in a later PR (when we're next in the area) as well.